io.github.Matteo842.SaveState.yml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.Matteo842.SaveState
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: SaveState-wrapper
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
+  - --filesystem=xdg-documents:rw
+  - --filesystem=~/.steam:rw
+  - --filesystem=~/.local/share/Steam:rw
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:rw
+  - --filesystem=~/snap:ro
+
+modules:
+  - name: savestate
+    buildsystem: simple
+    build-commands:
+      # 1. Creazione cartelle standard
+      - mkdir -p /app/bin /app/share/applications /app/share/icons/hicolor/256x256/apps /app/share/doc/savestate
+      
+      # 2. Installazione eseguibile (estrae dal tar.gz scaricato)
+      # Nota: Sostituisci 'SaveState.AppImage' col nome esatto che c'è nel tar.gz
+      - cp SaveState.AppImage /app/bin/SaveState-bin
+      - chmod +x /app/bin/SaveState-bin
+      
+      # 3. Installazione Documentazione e Licenza
+      - cp README.md LICENSE /app/share/doc/savestate/ || true
+      
+      # 4. Wrapper per evitare errori FUSE
+      - |
+        cat <<EOF > /app/bin/SaveState-wrapper
+        #!/bin/sh
+        exec /app/bin/SaveState-bin --appimage-extract-and-run "\$@"
+        EOF
+      - chmod +x /app/bin/SaveState-wrapper
+      
+      # 5. Icona e Menu
+      - cp io.github.Matteo842.SaveState.png /app/share/icons/hicolor/256x256/apps/
+      - |
+        cat <<EOF > /app/share/applications/io.github.Matteo842.SaveState.desktop
```

Comment:
Similar as above. 

The .desktop file should be upstream, not generated as an obscure shell hack.